pipeline/scraper/github.py

```diff
@@ -110,11 +110,7 @@ def run_query(today, language='JavaScript'):
                 last_node = result['data']['search']['edges'][-1]['cursor']
             else:
                 break
-        except ValueError as exc:
-            # pylint: disable=line-too-long
-            print(f"Could not run query starting at {last_node} for {daily_search_str}: {exc}: {result}")
-            break
-        except TypeError as exc:
+        except (ValueError, TypeError) as exc:
```

Comment:
TIL how to specify two exception types with a single statement, woot.

Reply:
@djsauble it's usually not a best practice though, i only did here since the message is the same and it makes it easy to cover with tests.